components/help_bar.py

```diff
@@ -0,0 +1,15 @@
+# In a file called [project root]/components/calendar/calendar.py
```

Comment:
Is this comment usefull?

Reply:
I saw it several times more in the code together with some other comments maybe get rid of those before we pollute the whole code base with it. ;)